### PR TITLE
Document missing character behavior when drawing text with fonts

### PR DESCRIPTION
**Pull request by Calinou**
Note that there's a [proposal](https://github.com/godotengine/godot-proposals/issues/591) about changing this behavior (which I think is better), but I don't know how difficult it is to implement.

See #40405.

### CODE REVIEW

**Review by akien-mga — APPROVED** [review]

File: doc/classes/Font.xml [hunk 1]
```diff
@@ -5,6 +5,7 @@
 	</brief_description>
 	<description>
 		Font contains a Unicode-compatible character set, as well as the ability to draw it with variable width, ascent, descent and kerning. For creating fonts from TTF files (or other font formats), see the editor support for fonts.
+		[b]Note:[/b] If a font doesn't contain a character used in a string, the character in question will be hidden without displaying any replacement character in the string.
```
Comment (bruvzg) [inline doc/classes/Font.xml:8]: `BitmapFont` will display nothing.
`DynamicFont` will try to display 0xFFFD if it is in font.

Comment (Calinou) [reply doc/classes/Font.xml:8]: > `DynamicFont` will try to display 0xFFFD if it is in font.

Weird, I thought that wasn't the case. It seems nobody can get a fallback character to display in practice. Should we change the character displayed or try another fallback character if `0xFFFD` isn't available?

Comment (bruvzg) [reply doc/classes/Font.xml:8]: Test project in the #40405 do display replacement character (diamond with a question mark), it just has Bold and Regular fonts with the different characters supported.

Comment (akien-mga) [reply doc/classes/Font.xml:8]: So should this change be updated to mention that a display replacement character will be shown *if supported by the font*?

Comment (akien-mga) [reply doc/classes/Font.xml:8]: Ping @Calinou @bruvzg.

Comment (bruvzg) [reply doc/classes/Font.xml:8]: With `BitmapFont` nothing will be displayed even if the `U+FFFD` character is supported by the font.

**Review by bruvzg — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.3.

**Pull request merged**